README.md

```diff
@@ -1,3 +1,57 @@
 # Project Keeper Maven Plugin
 
-* [Changelog](doc/changes/changelog.md)
\ No newline at end of file
+This maven plugin checks and unifies the project's structure according to the Exasol integration team's repository standards.
+
+## Installation
+
+Install this plugin by adding the following lines to your project's `pom.xml` file:
+
+```xml
+<plugin>
+    <groupId>com.exasol</groupId>
+    <artifactId>project-keeper-maven-plugin</artifactId>
+    <version>0.1.0</version>
+    <executions>
+        <execution>
+            <goals>
+                <goal>verify</goal>
+            </goals>
+        </execution>
+    </executions>
+    <configuration>
+        <modules>
+            <!-- add modules here: --> 
+            <!-- <module>For available modules see below.</module>-->
+        </modules>
+    </configuration>
+</plugin>
+```
+
+
+### Modules
+
+This plugin provides different template modules for different kinds of projects.
+
+* `default` (always included)
```

Comment:
What happened to the second one?

Reply:
I removed it since it was empty until now. I will add it again when it contains something